NonValidatingFactory missing from JDBC3 driver builds

Hi

A customer reported that they were unable to use the NonValidatingFactory with newer driver releases. Using a JDBC URL like jdbc:postgresql://ipaddress:port/dbname?ssl=true&sslfactory=org.postgresql.ssl.NonValidatingFactory would fail with an exception such as:

Caused by: java.lang.ClassNotFoundException: org.postgresql.ssl.NonValidatingFactory

This can be confirmed by compiling the following test program and running it against the JDBC3 driver. It'll fail on the JDBC3 driver but work fine with the same JDBC4 driver release.

```
import java.sql.Connection;
import java.sql.DriverManager;

public class JDBCConnect {
    public static void main(String[] args) throws Exception {
        Class.forName("org.postgresql.Driver");
        Connection con = DriverManager.getConnection(
                             "jdbc:postgresql:test?ssl=true&sslfactory=org.postgresql.ssl.NonValidatingFactory",
                             "test",
                             "password");
        System.out.println("OK!");

    }   
}
```

Result:

```
$ javac JDBCConnect.java && java -cp postgresql-9.2-1003.jdbc4.jar: JDBCConnect
jar:file:/home/craig/Downloads/postgresql-9.2-1003.jdbc4.jar!/org/postgresql/ssl/NonValidatingFactory.class
OK!

$ javac JDBCConnect.java && java -cp postgresql-9.2-1003.jdbc3.jar: JDBCConnect
Exception in thread "main" org.postgresql.util.PSQLException: The SSLSocketFactory class provided org.postgresql.ssl.NonValidatingFactory could not be instantiated.
        at org.postgresql.ssl.jdbc3.AbstractJdbc3MakeSSL.convert(AbstractJdbc3MakeSSL.java:58)
        .... elided ....
Caused by: java.lang.ClassNotFoundException: org.postgresql.ssl.NonValidatingFactory
        at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
        .... elided ...
```

It looks like it's an issue with `build.xml`'s target selection. I'm looking into it now.

--- a/build.xml
+++ b/build.xml
@@ -85,6 +85,7 @@
     <!-- ssl -->
     <include name="${package}/ssl/jdbc4/*.java" if="jdbc4any"/>
     <include name="${package}/ssl/jdbc3/*.java" if="jdbc3any"/>
+    <include name="${package}/ssl/.java" if="jdbc3any"/>
 
     <!-- gss -->
     <include name="${package}/gss/*.java"/>
@@ -249,6 +250,7 @@
        <!-- ssl -->
        <include name="${package}/ssl/jdbc4/*.java" if="jdbc4any"/>
        <include name="${package}/ssl/jdbc3/*.java" if="jdbc3any"/>
+       <include name="${package}/ssl/*.java" if="jdbc3any"/>
 
        <!-- gss -->
        <include name="${package}/gss/*.java" />

